File: src/main/java/aww/yeah/TwoHundredApplication.java
package aww.yeah;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;

import java.util.Random;

@SpringBootApplication
@Configuration
public class TwoHundredApplication {
    public static void main(String[] args) {
        SpringApplication.run(TwoHundredApplication.class, args);
    }

    @Bean
    public Random getRandom(){
        return new Random();
    }
}

File: src/main/kotlin/aww/yeah/TwoHundredApplication.kt
package aww.yeah

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.stereotype.Service
import java.util.Random

@SpringBootApplication
class TwoHundredApplication

fun main(args: Array<String>) {
    runApplication<TwoHundredApplication>(*args)
}

@Service
class ShowRandomGif(private val random: Random = Random()): () -> Boolean {
    override fun invoke(): Boolean =
            random.nextInt(5) == 0

}
